cypress/e2e/models/administration/proxy/proxy.ts

```
@@ -0,0 +1,121 @@
+    submitForm,
+} from "../../../../utils/utils";
+import { administration, button, SEC } from "../../../types/constants";
+import { CredentialsProxyData, ProxyData } from "../../../types/types";
+import {
+    excludedList,
+    navLink,
+    ProxyType,
+    ProxyViewSelectorsByType,
+} from "../../../views/proxy.view";
+
+export class Proxy {
```

Biome (2.1.2)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)